client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import sitesList from 'lib/sites-list';
+import PlanThankYouCard from 'blocks/plan-thank-you-card';
+
+const sites = sitesList();
+
+const PlanThankYouCardExample = React.createClass( {
```

Comment:
Done. I have used example.jsx from some other block that apparently was little outdated